src/CheckoutSdk/Payments/Sender/PaymentIndividualSender.cs

```diff
@@ -9,15 +9,17 @@ public PaymentIndividualSender() : base(PaymentSenderType.Individual)
         }
 
         public string FirstName { get; set; }
+        
+        public string MiddleName { get; set; }
 
         public string LastName { get; set; }
+        
+        public string Dob { get; set; }
 
         public Address Address { get; set; }
 
         public AccountHolderIdentification AccountHolderIdentification { get; set; }
         
-        public string MiddleName { get; set; }
-
         public string ReferenceType { get; set; }
 
         public string DateOfBirth { get; set; }
```

Comment:
DateOfBird is Dob, no? Probably it must be deleted

Reply:
Each belongs to different version, `dob` for CS2 and `date_of_birth` for CS1, that's why we keep both